packages/fluentui/react-northstar/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -26,7 +26,7 @@ export const radioGroupItemStyles: ComponentSlotStylesPrepared<RadioGroupItemSty
     borderStyle: 'solid',
     borderWidth: `${pxToRem(1)}`,
     borderColor: 'transparent',
-    borderRadius: `${pxToRem(2)}`,
+    borderRadius: siteVariables.borderRadius,
```

Comment:
Resolve if this is is intentionally not `borderRadiusSmall`
```suggestion
    borderRadius: siteVariables.borderRadiusSmall,
```

Reply:
It's the focus border for Radio Group:
![image](https://user-images.githubusercontent.com/6237759/115473609-def8c380-a1f0-11eb-849b-f55f9c210c59.png)

before:
![image](https://user-images.githubusercontent.com/6237759/115473641-ea4bef00-a1f0-11eb-8395-72dc42cdeb99.png)